packages/components/.babelrc

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
Do we need to have separate `.babelrc` and `.eslintrc` files for each package? Is there any way to have a single of each rc file at the root level?

Reply:
https://github.com/udacity/fluent-react-utils/issues/14